x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -200,10 +203,11 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
         container={document.querySelector<HTMLElement>('.application') || undefined}
         button={
           <DragDrop
+            noKeyboardSupportYet
+            draggable
             label={field.displayName}
```

Comment:
```suggestion
            label={i18n.translate('xpack.lens.indexPattern.fieldStatsDragButtonAriaLabel', {
              defaultMessage: 'Reorder {fieldName}',
              values: { fieldName: field.displayName },
            })}
```

Reply:
hmm, that doesn't seem right to me – this is the left datapanel and the fields cannot be reordered - they can be only copied to the workspace or in the dimension panel...

Reply:
Ah, my bad! Ignore this!